### PR TITLE
core: remove estimated-input-latency and first-cpu-idle

### PR DESCRIPTION
**Pull request by connorjclark**
ref #11866

### CODE REVIEW

File: lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js [hunk 1]
```diff
@@ -18,9 +18,10 @@ module.exports = [
         'first-contentful-paint': {
           numericValue: '>2000',
         },
-        'first-cpu-idle': {
-          numericValue: '>2000',
-        },
+        // TODO: ?
```
Comment (connorjclark) [inline lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js:21]: Should I replace these smoke expectations with something else?

**Comment by brendankenny** [comment]
what happened to `debug.json`?

**Comment by connorjclark** [comment]
I started just removing the EIL/FCI references then decided to just strip the entire thing but what is needed for treemap

**Comment by brendankenny** [comment]
could you split that and we could just land that quickly independently of other changes?

e.g. I updated individual lines in `debug.json` in #12554 and I'd rather just have that over and done with rather than waiting for this PR to land :)

File: lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js (hunk 1 above)
Comment (paulirish) [reply lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js:21]: nah. FCI is basically TTI and TTI is already there.

File: lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js [hunk 2]
```diff
@@ -26,9 +26,10 @@ module.exports = [
         'first-meaningful-paint': {
           score: '>=0.90', // primarily just making sure it didn't fail/go crazy, specific value isn't that important
         },
-        'first-cpu-idle': {
-          score: '>=0.90', // primarily just making sure it didn't fail/go crazy, specific value isn't that important
-        },
+        // TODO ?
```
Comment (paulirish) [inline lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js:29]: flagging to remove

File: lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js [hunk 3]
```diff
@@ -17,10 +17,11 @@ module.exports = [
       requestedUrl: 'http://localhost:10200/tricky-tti.html',
       finalUrl: 'http://localhost:10200/tricky-tti.html',
       audits: {
-        'first-cpu-idle': {
```
Comment (paulirish) [inline lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js:20]: on these you could add assertions for TBT if you want, but i don't think that's necessary or a  great test for that metric.

so i'm +1 on removing in this file too

**Review by paulirish — APPROVED** [review]
a few more things you'll see if you search `FCPUI`. 

There's a few errors that appear to be safe to delete. (they were never in the proto

And there's a block in `lantern-baseline-accuracy.json` that you'll need to delete.

**Comment by patrickhulce** [comment]
@paulirish has got this one under control 👍

File: lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js (hunk 1 above)
Comment (brendankenny) [reply lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js:21]: delete?

File: lighthouse-core/test/computed/metrics/estimated-input-latency-test.js [hunk 4]
```diff
@@ -1,69 +0,0 @@
...
-    it('uses a 5s rolling window', async () => {
-      const events = [
-        {start: 7500, end: 10000, duration: 2500},
-        {start: 10000, end: 15000, duration: 5000},
-      ];
-
-      assert.equal(EstimatedInputLatency.calculateRollingWindowEIL(events), 4516);
-    });
-
-    it('handles continuous tasks', async () => {
```
Comment (brendankenny) [inline lighthouse-core/test/computed/metrics/estimated-input-latency-test.js:48]: good stuff in here. Oh well :)

**Review by brendankenny — APPROVED** [review]
one last TODO to remove and looks like need to run `yarn update:test-devtools` one more time, but otherwise looks good to ship

nice work in all of those (many) test files

**Pull request merged**